api-client/package.json

```diff
@@ -22,12 +22,13 @@
   },
   "dependencies": {
     "@turf/distance": "^6.5.0",
+    "@types/luxon": "^3.3.0",
+    "intl-pluralrules": "^2.0.0",
```

Comment:
@steffenkleinle because there was an error [i18next::pluralResolver: Your environment seems not to be Intl API compatible, use an Intl.PluralRules polyfill](https://stackoverflow.com/questions/70493788/i18nextpluralresolver-your-environment-seems-not-to-be-intl-api-compatible-u)